site/template.html

```diff
@@ -12,5 +12,7 @@
 <body>
 {content}
 </body>
+<script src='/assembly/clipboard.min.js'></script>
```

Comment:
🤔 No more information really. I checked out this branch, did `yarn install`, then ran it, and it didn't work. That `clipboard.min.js` file was not being found by the browser. Are you sure everything's committed & building correctly? Could be that it got set up correctly for you but not reproducibly? Is it working for others?